### ISSUE TITLE
ShimGen only packer.exe, but packer finds *.exe.ignore files as plugins

### ISSUE THREAD

**Issue by StefanScherer** [StefanScherer/chocolatey-packages#5]
I want to add `*.exe.ignore` to the **packer** chocolatey package so Chocolatey only ShimGen the `packer.exe`.

But `packer.exe` 0.7.5 has a "bug" and finds all the `*.exe.ignore` files as plugins and overwrites the correct name of the exe. Starting such a plugin then fails:

```
2015/06/04 09:52:44 [INFO] Packer version: 0.7.5  9cd66feeacbd9cb318b72eb5ed59428c5b8c37ac+CHANGES
2015/06/04 09:52:44 Packer Target OS/Arch: windows amd64
2015/06/04 09:52:44 Built with Go Version: go1.3.3
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: amazon-chroot = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-amazon-chroot.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: amazon-chroot = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-amazon-chroot.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: amazon-ebs = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-amazon-ebs.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: amazon-ebs = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-amazon-ebs.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: amazon-instance = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-amazon-instance.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: amazon-instance = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-amazon-instance.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: digitalocean = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-digitalocean.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: digitalocean = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-digitalocean.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: docker = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-docker.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: docker = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-docker.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: googlecompute = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-googlecompute.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: googlecompute = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-googlecompute.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: null = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-null.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: null = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-null.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: openstack = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-openstack.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: openstack = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-openstack.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: parallels-iso = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-parallels-iso.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: parallels-iso = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-parallels-iso.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: parallels-pvm = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-parallels-pvm.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: parallels-pvm = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-parallels-pvm.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: qemu = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-qemu.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: qemu = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-qemu.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: virtualbox-iso = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-virtualbox-iso.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: virtualbox-iso = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-virtualbox-iso.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: virtualbox-ovf = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-virtualbox-ovf.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: virtualbox-ovf = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-virtualbox-ovf.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: vmware-iso = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-vmware-iso.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: vmware-iso = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-vmware-iso.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: vmware-vmx = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-vmware-vmx.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: vmware-vmx = C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-vmware-vmx.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: atlas = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-atlas.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: atlas = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-atlas.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: compress = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-compress.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: compress = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-compress.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: docker-import = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-docker-import.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: docker-import = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-docker-import.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: docker-push = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-docker-push.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: docker-push = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-docker-push.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: docker-save = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-docker-save.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: docker-save = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-docker-save.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: docker-tag = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-docker-tag.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: docker-tag = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-docker-tag.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: vagrant-cloud = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-vagrant-cloud.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: vagrant-cloud = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-vagrant-cloud.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: vagrant-vmware-ovf = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-vagrant-vmware-ovf.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: vagrant = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-vagrant.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: vagrant = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-vagrant.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: vcloud = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-vcloud.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: vsphere = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-vsphere.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: vsphere = C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-vsphere.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: ansible-local = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-ansible-local.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: ansible-local = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-ansible-local.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: chef-client = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-chef-client.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: chef-client = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-chef-client.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: chef-solo = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-chef-solo.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: chef-solo = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-chef-solo.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: file = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-file.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: file = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-file.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: puppet-masterless = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-puppet-masterless.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: puppet-masterless = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-puppet-masterless.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: puppet-server = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-puppet-server.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: puppet-server = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-puppet-server.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: salt-masterless = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-salt-masterless.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: salt-masterless = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-salt-masterless.exe.ignore
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: shell = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-shell.exe
2015/06/04 09:52:44 [DEBUG] Discoverd plugin: shell = C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-shell.exe.ignore
2015/06/04 09:52:44 Attempting to open config file: C:\Users\vagrant\AppData\Roaming\packer.config
2015/06/04 09:52:44 File doesn't exist, but doesn't need to. Ignoring.
2015/06/04 09:52:44 Packer config: &{DisableCheckpoint:false DisableCheckpointSignature:false PluginMinPort:10000 PluginMaxPort:25000 Builders:map[parallels-iso:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-parallels-iso.exe.ignore parallels-pvm:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-parallels-pvm.exe.ignore vmware-vmx:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-vmware-vmx.exe.ignore amazon-chroot:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-amazon-chroot.exe.ignore amazon-instance:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-amazon-instance.exe.ignore digitalocean:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-digitalocean.exe.ignore googlecompute:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-googlecompute.exe.ignore openstack:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-openstack.exe.ignore amazon-ebs:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-amazon-ebs.exe.ignore null:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-null.exe.ignore docker:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-docker.exe.ignore qemu:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-qemu.exe.ignore virtualbox-iso:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-virtualbox-iso.exe.ignore virtualbox-ovf:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-virtualbox-ovf.exe.ignore vmware-iso:C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-vmware-iso.exe.ignore] PostProcessors:map[compress:C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-compress.exe.ignore docker-import:C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-docker-import.exe.ignore docker-save:C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-docker-save.exe.ignore vagrant-vmware-ovf:C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-vagrant-vmware-ovf.exe.ignore vcloud:C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-vcloud.exe.ignore vsphere:C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-vsphere.exe.ignore atlas:C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-atlas.exe.ignore docker-push:C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-docker-push.exe.ignore docker-tag:C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-docker-tag.exe.ignore vagrant-cloud:C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-vagrant-cloud.exe.ignore vagrant:C:\ProgramData\chocolatey\lib\packer\tools\packer-post-processor-vagrant.exe.ignore] Provisioners:map[chef-solo:C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-chef-solo.exe.ignore file:C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-file.exe.ignore puppet-masterless:C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-puppet-masterless.exe.ignore puppet-server:C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-puppet-server.exe.ignore salt-masterless:C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-salt-masterless.exe.ignore shell:C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-shell.exe.ignore ansible-local:C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-ansible-local.exe.ignore chef-client:C:\ProgramData\chocolatey\lib\packer\tools\packer-provisioner-chef-client.exe.ignore]}
2015/06/04 09:52:44 Setting cache directory: C:\Users\vagrant\Documents\ubuntu-vm\packer_cache
2015/06/04 09:52:44 Reading template: ubuntu1204.json
2015/06/04 09:52:44 Skipping build 'vmware-iso' because not specified by -only.
2015/06/04 09:52:44 Creating build: virtualbox-iso
2015/06/04 09:52:44 Loading builder: virtualbox-iso
2015/06/04 09:52:44 Creating plugin client for path: C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-virtualbox-iso.exe.ignore
2015/06/04 09:52:44 Starting plugin: C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-virtualbox-iso.exe.ignore []string{"C:\\ProgramData\\chocolatey\\lib\\packer\\tools\\packer-builder-virtualbox-iso.exe.ignore"}
2015/06/04 09:52:44 ui error: Failed to create build 'virtualbox-iso': 

fork/exec C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-virtualbox-iso.exe.ignore: %1 is not a valid Win32 application.
2015/06/04 09:52:44 waiting for all plugin processes to complete...
```

**Comment by StefanScherer** [comment]
I have created a PR for packer at https://github.com/mitchellh/packer/pull/2173

**Comment by StefanScherer** [comment]
I have created a PR #6 to add these `*.ignore` files to the Chocolatey package when packer can skip these files.

**Comment by StefanScherer** [comment]
fixed with #6, packer 0.8.0
